guides/doc-Release_Notes/master.adoc

```diff
@@ -10,6 +10,7 @@ ifdef::katello[]
 
 include::topics/foreman.adoc[leveloffset=+1]
 include::topics/katello.adoc[leveloffset=+1]
+include::topics/katello-4.7.0.adoc[leveloffset=+1]
```

Comment:
I'd at least visually separate this bit here. Mostly because later on we're going to interweave versions. And I checked, but nobody did this right so perhaps the README is too unclear?

I opened https://github.com/theforeman/foreman-documentation/pull/1787 with a hint of where it should be inserted.

Originally I thought it's be chronologically and Katello is always released after Foreman, so it's always:
* Katello x.y.0
* Foreman a.b.0

Now I wonder if users prefer reading the Katello notes before Foreman, or the other way around.

Reply:
Rebased on the upstream branch and added them under Foreman in the new spot